pages/jdl.md

```diff
@@ -1,1447 +0,0 @@
-The resulting application will have:
-  - baseName: `jhipster`
-  - applicationType: `monolith`
-  - databaseType: `sql`
-  - etc.
-
-Now, if you want some custom options:
-
-```
-application {
-  config {
-    baseName myapp
-    applicationType microservice
-    prodDatabaseType postgresql
-    buildTool gradle
-  }
-}
-```
-
-Those options are only a sample of what's available in the JDL.
-The complete list of options is available in the annexes, [here](#annexes).
-
-If you want more than one application, here's how you do it:
-
-```
-application { // will be generated under the 'myFirstApp' folder
-  config {
-    baseName myFirstApp
-  }
-}
-
-application { // will be generated under the 'mySecondApp' folder
-  config {
-    baseName mySecondApp
-    applicationType microservice
-  }
-}
-```
-
-You can have as many application as you want in as many file as you wish: there's no limitation.
-
-Declaring entities is the most basic thing available and now you can set what entity should be generated
-in the applications you want.
-
-Let's improve the previous example:
-
-```
-application {
-  config {
-    baseName myMonolith
-    applicationType monolith
-  }
-  entities * except C, D
-}
-
-application {
-  config {
-    baseName myGateway
-    applicationType gateway
-    serverPort 9042
-  }
-  entities * except A, B
-}
-
-application {
-  config {
-    baseName microserviceA
-    applicationType microservice
-  }
-  entities C
-}
-
-application {
-  config {
-    baseName microserviceB
-    applicationType microservice
-    serverPort 8082
-  }
-  entities D
-}
-
-entity A
-entity B
-entity C
-entity D
-
-dto * with mapstruct
-service * with serviceClass
-paginate D with pager
-```
-
-Now, several things will happen when generating these applications and folders:
-  - Four applications will be created:
-    - myMonolith in `./myMonolith`, with the server port `8080`
-    - myGateway in `./myGateway`, with the server port `9042`
-    - microserviceA in `./microserviceA`, with the server port `8081`
-      - Even though we didn't specify a server port, JHipster sets one by default.
-      - For microservices, the default one is `8081`
-      - For gateways and monoliths, it's `8080`
-      - For UAA apps, it's `9999`
-    - microserviceB in `./microserviceB` with the server port `8082`
-  - Four entities will be generated
-    - `A` and `B` in the monolith
-    - `C` and `D` both in the gateway
-      - `C` in the first microservice
-      - `D` in the second microservice
-  - The `microservice` option is implicit for `C` and `D`
-    - Because they get generated on the two microservices, this option will be set by default.
-  - Options work the same way as before
-
-Note that the generator sets default values if they aren't present (like the `databaseType`).
-JHipster Core does the exact same things.
-
-### <a name="entitydeclaration"></a> Entity declaration
-
-The entity declaration is done as follows:
-
-    [<entity javadoc>]
```

Comment:
This meta syntax that we had in each section was useful for advanced users, so may be we can retain this in the individual sections. WDYT?

Reply:
I removed them because I felt having them was redundant as examples are introduced. But maybe you're right